File: agent-connect-android/app/src/main/java/sg/searchhouse/agentconnect/model/api/community/CommunityPostMediaPO.kt
package sg.searchhouse.agentconnect.model.api.community

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Doc: https://streetsine.atlassian.net/wiki/spaces/SIN/pages/1140719619/Communities+Data+Structure
 */
data class CommunityPostMediaPO(
    @SerializedName("id")
    val id: Long,
    @SerializedName("type")
    val type: Int,
    @SerializedName("mediaType")
    val mediaType: String,
    @SerializedName("url")
    val url: String,
    @SerializedName("urlThumbnail")
    val urlThumbnail: String,
    @SerializedName("dateCreate")
    val dateCreate: String
) : Serializable

File: srxproperty/src/dataObject/CommunityPostMediaPO.js
import {ObjectUtil} from '../utils';

class CommunityPostMediaPO {
  constructor(data) {
    if (!ObjectUtil.isEmpty(data)) {
      this.id = data.id;
      this.type = data.type;
      this.mediaType = data.mediaType;
      this.url = data.url;
      this.thumbnailUrl = data.urlThumbnail;
      this.dateCreate = data.dateCreate;
      this.postId = data.postId;
    }
  }
}

export {CommunityPostMediaPO};
